### ISSUE TITLE
How to feed values for dropout node?

### ISSUE THREAD

**Issue by pallavi7naik** [tensorflow/lucid#120]
Hi. as seen in the picture below, to calculate relevance values i need to feed a value to the placeholder "keep_prob" . in tf i would feed it as : 

**sess.run(train_step, feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})**

how do i feed value in lucid? [
![screenshot from 2018-10-30 14-25-58](https://user-images.githubusercontent.com/39490600/47706773-186b9c00-dc50-11e8-87b2-4f1c32141287.png)
](url)

**Comment by hegman12** [comment]
Hi,
typically, an inference model should not have dropout node as required adjustment(dividing the activation by keep probability) would have been done at training time, i.e. dropout is actually an inverse dropout. If you still have to pass keep_prob, [see this line of code and add one more if condition to retrieve your dropout node](https://github.com/tensorflow/lucid/blob/d347d4e0d50a9ce514900a258cd850970f6fbbaa/lucid/optvis/render.py#L258)

then you can use `optiviz.render.make_viz_T` function to actually optimize and pass the dropout value during optimization.
Note: please try and see if this works, i have not tested change.

**Comment by ludwigschubert** [comment]
@hegman12 thanks for replying; sounds correct to me. (We will aim at having more examples of using lucid without `render.render` in the future.)
